src/environments/array.js

```diff
@@ -67,13 +69,19 @@ function parseArray(
         arraystretch?: number,
         colSeparationType?: ColSeparationType,
         addEqnNum?: boolean,
+        maxNumRows?: 1,
+        maxNumCols?: 1 | 2,
```

Comment:
```suggestion
        maxNumRows?: number,
        maxNumCols?: number,
```
I see no reason to limit them to 1 or 2.

Reply:
My reasoning was:
* `{equation}` is limited to one column
* `{split}` is limited to no more than two columns
* Every other environment has no limit.

So an environment can either pass on putting `maxNumCols` into the payload, or specify 1 or 2. Anything else makes no sense.

Reply:
Doesn't `{array}` have `maxNumCols` of the number of its alignment specifiers?

Reply:
You know, that would be a good thing to write into the KaTeX code for `{array}`. It is not there now. And then your point about `maxNumCols` type makes perfect sense. I'll do it.

Reply:
For `maxNumRows`, I cannot think of any use case that limits to more than one. If that's the case, I think we can make it a boolean, e.g., `singleRow`.

Reply:
Done.